src/wallet/test/CMakeLists.txt

```diff
@@ -10,6 +10,7 @@ target_sources(test_bitcoin
     wallet_test_fixture.cpp
     db_tests.cpp
     coinselector_tests.cpp
+    coinselection_tests.cpp
```

Comment:
nit: Since `coinselection_tests.cpp` is introduced in the first commit, this change may be there as well.